File: xFit/UI/xfit_admin/lib/models/clanska_karta.dart

import 'package:json_annotation/json_annotation.dart';
import 'korisnik.dart';

part 'clanska_karta.g.dart';

@JsonSerializable()

class ClanskaKarta {
  int? clanskaKartaId;
  String? sadrzaj;
  int? korisnikId;
  Korisnik? korisnik;

  ClanskaKarta(this.clanskaKartaId, this.sadrzaj, this.korisnikId, this.korisnik);

  factory ClanskaKarta.fromJson(Map<String, dynamic> json) => _$ClanskaKartaFromJson(json);

  Map<String, dynamic> toJson() => _$ClanskaKartaToJson(this);
}

File: xFit_rabbit/xFit_rabbit/Data/ClanskaKarta.cs
using System;
using System.Collections.Generic;

namespace xFit_rabbit.Data;

public partial class ClanskaKarta
{
    public int ClanskaKArtaId { get; set; }

    public string? Sadrzaj { get; set; }

    public int? KorisnikId { get; set; }

    public virtual Korisnik? Korisnik { get; set; }
}
